File: ui-service/src/app/model/user.model.ts
export class UserModel {
  public identityNumber: string;
  public name: string;
  public surname: string;
  public phone: string;
  public city: string;
  public score: bigint;

  constructor(identityNumber: string, name: string, surname: string, phone: string, city: string, score: bigint) {
    this.identityNumber = identityNumber;
    this.name = name;
    this.surname = surname;
    this.phone = phone;
    this.city = city;
    this.score = score;
  }

}

File: api-service/src/main/java/com/koc/finans/api/service/model/UserModel.java
package com.koc.finans.api.service.model;

import lombok.Getter;
import lombok.Setter;

import javax.persistence.*;

@Entity
@Table(name = "user_score")
@Getter
@Setter
public class UserModel {
    @Id
    @Column(name = "identity_number", nullable = false)
    protected String identityNumber;
    @Column(name = "phone", nullable = false)
    protected String phone;
    @Column(name = "name", nullable = false)
    protected String name;
    @Column(name = "surname", nullable = false)
    protected String surname;
    @Column(name = "city", nullable = false)
    protected String city;
    @Column(name = "score", nullable = false)
    protected int score;

}
